src/components/TodoFooter/TodoFooter.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+
+import TodoFilter from '../TodoFilter';
+
+import { deleteTodo } from '../../api/todos';
+import { useTodos } from '../../hooks/useTodos';
+import { Todo } from '../../types';
+
+const Footer: React.FC = () => {
+  // eslint-disable-next-line prettier/prettier
+  const {
+    todos,
+    setTodos,
+    filter,
+    setFilter,
+    setIsAllDeleted,
+    handleError
+  } = useTodos();
+
+  const isClearButtonVisible = todos.some(todo => todo.completed);
+  const amountItemsLeft = todos.filter(todo => !todo.completed).length;
+
+  const handleDeletetodo = (todoId: number) => {
+    setIsAllDeleted(true);
```

Comment:
Do you delete only one todo or all here? Looks like this state should be `isLoading`